### PR TITLE
Second round of revisions, with a catchier title

### PR DESCRIPTION
**Pull request by rbanks54**

### CODE REVIEW

File: content/post/healthy-backlogs.md [hunk 1]
```diff
@@ -1,40 +1,69 @@
...
 
-## Dynamic
-Finally, a healthy Product Backlog has to be dynamic. A Product Owner isn't a Product Backlog Owner. A Product Owner has to trust the team so that any member is able to modify items in the backlog, however, it is the Product Owner's responsibility to order the backlog to maximise value from the team.
+Similarly, being open to change includes removing items. In long-lived backlogs it's not unusual to have many items which are no longer relevant, where the original context has been forgotten, or where the potential value has eroded over time. Items that are poorly worded, not transparent, are simply not understandable by the people that are now on the team. Don't be afraid to deleting these items. Keep the weeds out of the garden!
```
Comment (RaphHaddad) [inline content/post/healthy-backlogs.md]: don't be afraid to "delete" not deleting

**Comment by RaphHaddad** [comment]
I will add comments for me but I will merge . thanks @rbanks54

File: content/post/healthy-backlogs.md [hunk 2]
```diff
@@ -1,40 +1,69 @@
...
+
+They need to be well understood, reasonably sized, and ready to be worked on. This ensures there can be a steady flow of work pulled into the development team. These attributes aid the overall backlog health as explained below:
+
+#### Steady flow of work
+The product backlog needs to have enough refined items to cover the next 2 or 3 sprints, ensuring the delivery team has enough work and won't be delayed by trying to work out what can be worked on next.
+
+This also provides enough slack and freedom for the Product Owner to (re)order the backlog to maximise value across the upcoming sprints. Where there's only enough refined work for the next sprint, the Product Owner has no flexibility or capacity to respond to change should the situation require it (for example: risk)  
+
+#### Size
+When items are too large to be delivered completely within a sprint they are too large.
```
Comment (RaphHaddad) [inline content/post/healthy-backlogs.md]: When items are too large to be delivered completely within a sprint they are too large to be in the backlog.

Comment (rbanks54) [reply content/post/healthy-backlogs.md]: Be careful with that statement. Large items in a backlog are OK. They just shouldn't be near the top.

**Pull request merged**